src/pages/settings/Report/RoomNamePage.tsx

```diff
@@ -78,7 +75,7 @@ function RoomNamePage({report, policy, reports}: RoomNamePageProps) {
             includeSafeAreaPaddingBottom={false}
             testID={RoomNamePage.displayName}
         >
-            <FullPageNotFoundView shouldShow={ReportUtils.shouldDisableRename(report, policy)}>
+            <FullPageNotFoundView shouldShow={ReportUtils.shouldDisableRename(report)}>
```

Comment:
Why do we need this change?

Reply:
This was a change proposed in the linked PR regarding [moving the name section out of Settings](https://github.com/Expensify/App/pull/43636), you have the answer from the author [here](https://github.com/Expensify/App/pull/43636#issuecomment-2165435802).